airbyte-integrations/connectors/source-pendo-python/source_pendo_python/streams.py

```diff
@@ -185,10 +185,36 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 class Report(PendoPythonStream):
     name = "report"
 
+    @property
+    def use_cache(self) -> bool:
+        return True
+
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return "report"
 
 
+class ReportResult(HttpSubStream, PendoPythonStream):
+    name = "report_result"
+    primary_key = "reportId"
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"report/{stream_slice['parent']['id']}/results.json"
+
+    def parse_response(
+        self,
+        response: requests.Response,
+        stream_state: Mapping[str, Any],
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping]:
+        for record in response.json():  # GitHub puts records in an array.
```

Comment:
Should this comment be about Pendo?

Reply:
Good catch, I was learning about how to work with substreams by looking at the github connector